packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -11,15 +11,36 @@ import { getContractPath } from 'App/Components/Routes/helpers';
 import { website_name } from 'App/Constants/app-config';
 import { getSupportedContracts } from 'Constants';
 import { connect } from 'Stores/connect';
+import { WS } from 'Services/ws-methods';
 import { getStatementTableColumnsTemplate } from '../Constants/data-table-constants';
 import PlaceholderComponent from '../Components/placeholder-component.jsx';
 import { ReportsMeta } from '../Components/reports-meta.jsx';
 import EmptyTradeHistoryMessage from '../Components/empty-trade-history-message.jsx';
 import Shortcode from '../Helpers/shortcode';
+import 'Sass/app/modules/statement.scss';
 
 class Statement extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { total_deposits: 0, total_withdrawals: 0 };
+    }
```

Comment:
Instead of adding a constructor and state to a class component, can we change it to a functional component and use hooks instead?

Reply:
As per our discussion, we will do it later. Thanks.